lib/l10n/app_es.arb

```
@@ -287,8 +287,8 @@
   "deleteAllDataWarning": "Esto eliminará permanentemente todos tus chats, mensajes y configuraciones de este dispositivo. Esta acción no se puede deshacer.",
   "deleteAllDataError": "No se pudieron eliminar todos los datos. Por favor, inténtalo de nuevo.",
   "chatInformation": "Información del chat",
-  "addAsContact": "Añadir como contacto",
-  "removeAsContact": "Eliminar como contacto",
+  "follow": "Seguir",
```

RuboCop (1.86.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)